### PR TITLE
Beta micro site

### LINKED ISSUE #2650: Add Links to Privacy Statement and ToS
## Feature Description

As someone interested in Web Stories, I want to be able to view the Privacy statement and ToS.

Note:
Do something similar to Site Kit

---

_Do not alter or remove anything below. The following sections will be managed by moderators only._

## Acceptance Criteria

1. On the plugin download page / information page, a link to the Privacy statement is provided https://policies.google.com/privacy
2. On the plugin download page / information page, a link to the ToS is provided: https://policies.google.com/terms

### PR DESCRIPTION
**Pull request by pbakaus**
## Summary

Adds the micro site for the beta in Github Pages under /beta.

## To-do

- There's a placeholder in the top section that @samitron7 still needs to provide.
- Need to wait for @swissspidy to generate release to replace download link placeholders.
- CSS clean up. General markup structure should be pretty sound, but the CSS is rapidly coded. Then again, it's a site that will only exist a few weeks..

---
localhost_web-stories-wp_beta_

Fixes #2650

### CODE REVIEW

**Comment by swissspidy** [comment]
Really nice work! Just please hold off on merging this until I change the GitHub pages deployment (which I aim to do today). Otherwise it will be overridden with the next deployment of the storybook.

**Comment by pbakaus** [comment]
> Really nice work! Just please hold off on merging this until I change the GitHub pages deployment (which I aim to do today). Otherwise it will be overridden with the next deployment of the storybook.

Thanks! AFAIK though there should be no overwrite, as in our travis config, `keep_history` is set to `true`, thus not force pushing to gh-pages. I haven't tested it, but building storybook should not wipe this.

**Comment by pbakaus** [comment]
Alright, getting too sleepy now but managed to squeeze in fluidity on Desktop and a mobile breakpoint for common mobile devices.

**Comment by swissspidy** [comment]
I have tested it / I ran into it in the past 🙃  It does no force push thanks to `keep_history`, but because the storybook is in the root folder, it will override everything in the root. See my manual commit https://github.com/google/web-stories-wp/commit/1010aba30a8457e468e6f51823b1d3c79efafcc9 and then the override by the deployment https://github.com/google/web-stories-wp/commit/aa820a6f3ad06e5bda99d965a2d2a31fc9e1b6a3. But I will double check again.

**Comment by swissspidy** [comment]
@pbakaus FYI fixed the deployment part and made a small config change. Make sure to pull/rebase.

File: beta/index.html [hunk 1]
```diff
@@ -0,0 +1,627 @@
...
+                    <li>Navigate to Plugins &gt; Add New.</li>
+                    <li>Click the Upload Plugin button at the top of the screen.</li>
+                    <li>Select the zip file from your local filesystem.</li>
+                    <li>Click the Install Now button.</li>
+                    <li>When installation is complete, you’ll see “Plugin installed successfully.” Click the Activate Plugin button at the bottom of the page.</li>
+                </ol>
+            </dd>
+        
+            <dt>I found a bug or missing feature! How do I report it?</dt>
+            <dd>Awesome! That’s exactly what the beta is for. Please submit feedback via <a href="https://docs.google.com/forms/d/e/1FAIpQLSfRK9gFWMavvGbEWPFUFz8PDgrDE6XdVuh3B9e1Fu549V8BxQ/viewform">this feedback form</a> or <a href="https://github.com/google/web-stories-wp/issues">file a bug or feature request on Github</a>. Your help is greatly appreciated.</dd>
```
Comment (swissspidy) [inline beta/index.html]: Would be nice to use `#3474E0` for the color of these links too.

Comment (pbakaus) [reply beta/index.html]: Good catch, done

File: beta/index.html [hunk 2]
```diff
@@ -0,0 +1,905 @@
...
+                    <li>Navigate to Plugins &gt; Add New.</li>
+                    <li>Click the Upload Plugin button at the top of the screen.</li>
+                    <li>Select the zip file from your local filesystem.</li>
+                    <li>Click the Install Now button.</li>
+                    <li>When installation is complete, you’ll see “Plugin installed successfully.” Click the Activate Plugin button at the bottom of the page.</li>
+                </ol>
+            </dd>
+        
+            <dt>I found a bug or missing feature! How do I report it?</dt>
+            <dd>Awesome! That’s exactly what the beta is for. Please submit feedback and <a href="https://github.com/google/web-stories-wp/issues">file a bug or feature request on Github</a> for now - we'll follow up with a easier-to-use feedback form in the next days. Your help is greatly appreciated.</dd>
```
Comment (swissspidy) [inline beta/index.html]:
```suggestion
            <dd>Awesome! That’s exactly what the beta is for. Please submit feedback and <a href="https://github.com/google/web-stories-wp/issues">file a bug or feature request on Github</a> for now - we'll follow up with an easier-to-use feedback form in the next days. Your help is greatly appreciated.</dd>
```

**Review by swissspidy — APPROVED** [review]
Ship it!

**Pull request merged**